static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {Fragment} from 'react';
```

Comment:
I don't think you actually need to import `React` here.

Will figure out how to lint for this soon 😱